uiowa/uiowa
# Feature: Adding missing fields for AoS program types

Resolves https://github.com/uiowa/uiowa/issues/3001. 

# How to test

- `blt ds --site=admissions.uiowa.edu`
- `yarn workspace admissions_core build`
- `drush @admissions.local cr`
- Go to http://admissions.local.drupal.uiowa.edu/node/add/area_of_study and verify that the new fields are showing and show/hide based on checkbox. 

<img width="1197" alt="Screen Shot 2021-02-02 at 4 32 47 PM" src="https://user-images.githubusercontent.com/1036433/106671830-af80d600-6574-11eb-90cc-4f7071beaf03.png">
<img width="1258" alt="Screen Shot 2021-02-02 at 4 32 58 PM" src="https://user-images.githubusercontent.com/1036433/106671836-b14a9980-6574-11eb-8188-0d25b5474f91.png">



- Go to http://admissions.local.drupal.uiowa.edu/areas-of-study/accounting and verify that sidebar block is showing up with a new option like "Teaching Licensure" with a checkbox icon. 

<img width="440" alt="Screen Shot 2021-02-02 at 4 37 07 PM" src="https://user-images.githubusercontent.com/1036433/106671976-ebb43680-6574-11eb-86e3-364c7697b929.png">
